experimental/IntersectionTheory/src/blowup.jl

```diff
@@ -14,7 +14,7 @@ More precisely, return a tuple `(Bl, E, j)`, say, where
 
 # Examples
 
-Taken from the sage package Chow by Lehn/Sorger:
+# Taken from the sage package Chow by Lehn/Sorger:
```

Comment:
This introduces a new title just below the title "Examples", and both title are of the same size. I wonder if you truly desired this @wdecker . Left for you to fix/modify/alter if needed in another PR.